beetmoverscript/src/beetmoverscript/script.py

```diff
@@ -109,6 +109,26 @@ async def push_to_nightly(context):
     add_checksums_to_artifacts(context)
 
 
+# direct_push_to_releases {{{1
+async def direct_push_to_releases(context):
```

Comment:
@escapewindow curious what your thoughts are here. For Fenix, we'd likely push directly into ~releases, so no `candidates` so it didn't make sense to reuse that logic. On the other hand, the `push-to-candidates` also assumes balrog manifest logic + checksums which we don't really need here for now. So I thought adding a new action scope to reflect that since the `push-to-releases` is an existing different snippet.

Reply:
I'm ok with that. We may want to somehow list which trust domains support direct pushes, and which ones support candidates->releases, so we don't accidentally do the wrong thing in the wrong project.

Reply:
@escapewindow Resurrecting this thread - so it sounds like you're thinking of some additional check similar to https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L320 somewhere there but adjusted to cot product / actions?  CC @hneiva

Reply:
Yes, and/or baking it into the script config, e.g. iscript's [supported behaviors](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/iscript/config_example.yaml#L15-L20) allowlist.